types/index.d.ts

```diff
@@ -16,179 +16,174 @@ import {
   GraphQLTypeResolver,
 } from 'graphql';
 
-export = graphqlHTTP;
+export {};
```

Comment:
Why do we need this construct?

Reply:
dtslint was complaining about the types we don't export

> All declarations in this module are exported automatically. Prefer to explicitly write 'export' for clarity. If you have a good reason not to export this declaration, add 'export {}' to the module to shut off automatic exporting. See: https://github.com/Microsoft/dtslint/blob/master/docs/strict-export-declare-modifiers.md

Reply:
Wow, now I understand why it was complaining in `graphql-js`.